platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -47,28 +52,38 @@
 
   private static final Path BIN_DIR = Paths.get("bin");
 
+  private static final List<String> SYSTEM_PROPERTIES_TO_PRESERVE =
+      ImmutableList.of("solr.password");
+
+  private static final Path SYSTEM_PROPERTIES_PATH = Paths.get("etc", "custom.system.properties");
```

Comment:
Could this be changed to `CUSTOM_SYSTEM_PROPERTIES_PATH` and its respective usages (such as the `upgradeSystemProperties` method) to avoid confusion between the `custom.system.properties` file and the `system.properties` file? I think I have seen confusion with this before, so maybe this is how it should be? I'm not sure.

Reply:
Yes, I think that will help reduce confusion 😄